tests/Integration/Behaviour/behat.yml

```diff
@@ -288,83 +287,6 @@ default:
                 - Tests\Integration\Behaviour\Features\Context\CommonFeatureContext
                 - Tests\Integration\Behaviour\Features\Context\Domain\AttachmentFeatureContext
                 - Tests\Integration\Behaviour\Features\Context\LanguageFeatureContext
-        product:
```

Comment:
As we discussed earlier it's a bit too soon to unify the two suites, we'll do this in the next PR that cleans combination commands as well Maybe that's why the behat tests are failing for now because some steps are unknown

What you can do for now is keep the `product` and `update_product` suites, but clean the old context from the `product` suite